jupyterlab_caip_optimizer/src/components/trial_visualization.tsx

```diff
@@ -43,27 +62,36 @@ export interface TrialData {
   [key: string]: number | string;
 }
 
-// Needed for populating D3 axis
-export interface AxisData {
+/**
+ * AxisData needed for parallel coordinates.
+ * (PCP is short term for parallel coordinates)
+ */
+export interface PCPAxisData {
   label: string;
   type: string;
   minVal?: number;
   maxVal?: number;
   values?: string[] | number[];
 }
 
-function createAxesData(
-  axisLabelsLeft,
-  axisLabelsRight,
+/**
+ * Populates list of axis data for parallel coordinates.
+ * @param paramLabelList List of parameter names
+ * @param metricLabelList List of metric names
+ * @param axisPropsList List of axisProps
+ */
+function createPCPAxisDataList(
+  paramLabelList: string[],
+  metricLabelList: string[],
   axisPropsList: AxisPropsList
-): AxisData[] {
-  const axesData = [];
-  axisLabelsLeft.forEach(label => {
+): PCPAxisData[] {
+  const PCPAxisDataList = [];
```

Comment:
nit: I would keep this name lowercased (pcpAxisDataList?)

Reply:
Since it's interface, it would have to be uppercased, but would you prefer `PcpAxisDataList`? I thought all-caps would better indicate that it is an abbreviation

Reply:
fixed!